spacewalk/config/spacewalk-config.spec

```diff
@@ -54,6 +54,8 @@ Requires(preun): initscripts
 # We need package httpd to be able to assign group apache in files section
 Requires(pre): %{apachepkg}
 Requires:       openssl
+BuildRequires:  cobbler
+Requires:       cobbler
```

Comment:
This was not a good idea, because `spacewalk-config` is part of the client tools, where we don't want cobbler. In fact on CentOS/RES it doesn't work as cobbler is not available.

@mcalmer we need to talk on Monday about this.